src/main/webapp/aboutus.html

```diff
@@ -35,11 +35,11 @@ <h2>Anh Duc Le</h2>
       <li>Aspirational Hobby: Skiing </li>
       <li>Ask me About: Badminton</li>
     </ul>
-    <h2>Teammate B Name</h2>
+    <h2>Quang Minh Le</h2>
     <ul>
-      <li>Summer Feelz: </li>
-      <li>Aspirational Hobby: </li>
-      <li>Ask me About: </li>
+      <li>Summer Feelz: free, active and hot as hell </li>
+      <li>Aspirational Hobby: parachuting </li>
+      <li>Ask me About: One Piece </li>
```

Comment:
Oh really :)

Reply:
It was the first thing that came out to my mind!